python/whl.bzl

```diff
@@ -172,6 +206,11 @@ extract_wheels = repository_rule(
             default = Label("//tools:piptool.par"),
             cfg = "host",
         ),
+        "_patcher": attr.label(
```

Comment:
This is no longer needed?
Also, why is piptool.par being updated since you are not touching any python files?